tutorials/integrations/zapper.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Portfolio Tracking on Moonbeam with Zapper
+description: Learn how to track your holdings on Moonbeam using Zapper and interact with different DeFi platforms on the Moonbeam Ecosystem.
+---
+
+# Portfolio Tracking with Zapper
+
+## What is Zapper?
+
+[Zapper](https://zapper.xyz){target=\_blank} is a powerful Web3 application that allows users to explore on-chain data, manage portfolios, and interact with various decentralized finance (DeFi) protocols across multiple networks, including Moonbeam. By integrating with Zapper, users can seamlessly monitor and manage assets across Moonbeam's DeFi ecosystem, including Moonwell, StellaSwap, OmniLST, and Prime Protocol.
+
+This tutorial provides a detailed step-by-step guide for both users and developers to integrate with Zapper and maximize their experience within the Moonbeam ecosystem.
+
+## Getting Started on Zapper
+
+You can navigate to [Zapper's official website](https://zapper.xyz){target=\_blank} and take the following steps to start interacting with the application:
+
+- Click on the **Connect Wallet** button at the top right corner.
+
+![Zapper main site](/images/tutorials/integrations/Zapper/zapper-1.webp)
+
+- Select your preferred wallet from the supported options (e.g., MetaMask, Talisman, Coinbase Wallet).
+
+![Zapper wallet modal](/images/tutorials/integrations/Zapper/zapper-2.webp)
+
+- Follow the on-screen prompts to connect securely to Zapper.
+
+### Explore Your Portfolio
+
+Once your wallet is connected, Zapper will automatically detect and display your assets across supported networks, including Moonbeam. You can navigate to your main dashboard by clicking on **My Profile** on the left sidebar or selecting it from the wallet icon on the top-right corner. The dashboard provides an overview of your holdings and activity split in the following categories:
+
+- **Net Worth Summary**: Displays your total assets, liabilities, and net worth across all networks
+- **Token Balances**: Lists all tokens held in your connected wallet, including GLMR (Moonbeam's native token) and any ERC-20 tokens
+- **DeFi Positions**: Shows active positions in Moonbeam-based DeFi platforms such as lending, liquidity pools, and staking
+- **NFTs**: Displays any NFTs you own on Moonbeam or other supported chains
+- **On-Chain Activity**: Zapper's dashboard provides real-time data, enabling efficient management of your DeFi activities
+
+![Zapper Dashboard](/images/tutorials/integrations/Zapper/zapper-3.webp)
+
+### Engage with Moonbeam's DeFi Ecosystem
+
+Zapper provides access to view the activity on Moonbeam's DeFi protocols such as:
+
```

Comment:
Would be great to have the links for each of these protocols added - for Moonwell, StellaSwap, Omni, and Prime.